### ISSUE TITLE
Error on cookbook upload with open source server

### ISSUE THREAD

**Issue by Tech356** [xanzy/chef-guard#53]
When trying to upload a cookbook, the server returns a 502 error.

```
knife cookbook upload test --freeze
```

Chef-Guard.log

```
ERROR:   2014/10/15 15:39:31 Failed to dowload recipes/default.rb from the test cookbook: Get
https://chef.company.pvt/bookshelf/organization-00000000000000000000000000000000/checksum-3f4
1d00fdce2d290eee1983e2b85ba16?AWSAccessKeyId=5355300a9f7d0b348d0ff8986f38d5b20cd18895&Expires
=1413401981&Signature=PQ5OVgW9dSge5YqUq2DBHB5Nlw0%3D: x509: certificate signed by unknown authority
```

**Comment by svanharmelen** [comment]
The problem is that when retrieving cookbook files from the bookshelf CG is not checking if you want to verify SLL or not. So if you have self signed cert it will fail with this error whatever you have configured...

Will fix this asap and release a new version within the next day or so. Thx!

**Comment by svanharmelen** [comment]
Fixed by PR #55
